VimeoUpload/Upload/Controllers/VideoDeletionManager.swift

```diff
@@ -248,3 +246,9 @@ import VimeoNetworking
         self.operationQueue.isSuspended = !currentlyReachable
     }
 }
+
+private extension Selector {
```

Comment:
Nit: Can we bring this private extension to the top of the file?

Reply:
Since we're merging back from master we probably don't want to make any new changes here, right?